recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
Don't they have namespaces on Debian and Fedora? Anyone who's used AMPL before knows the ASL under that name so I agree with @tkralphs that changing it would be very confusing. In Homebrew, I made `asl` a `:recommended` dependency of `ipopt` because it's so useful and is a natural driver. I supposed it could be "libasl" but then, the ""Advanced Simulation Library" could also be "libasl".